From 37029b974f49a5b1c5072fa6bf040177c104f87f Mon Sep 17 00:00:00 2001
From: ghislain <ghislain@basetrend.nl>
Date: Fri, 11 Sep 2009 20:44:08 +0200
Subject: [PATCH] Enable keyboard in web brower for password fields.

Yesterday I found and repaired a little bug in the 'Web Browser': when
entering a password-field the keyboard doesn't show.
---
 .../webkit/WebCore/html/HTMLInputElement.cpp       |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/qtopiacore/qt/src/3rdparty/webkit/WebCore/html/HTMLInputElement.cpp b/qtopiacore/qt/src/3rdparty/webkit/WebCore/html/HTMLInputElement.cpp
index dae0be5..671e0f6 100644
--- a/qtopiacore/qt/src/3rdparty/webkit/WebCore/html/HTMLInputElement.cpp
+++ b/qtopiacore/qt/src/3rdparty/webkit/WebCore/html/HTMLInputElement.cpp
@@ -243,7 +243,7 @@ void HTMLInputElement::aboutToUnload()
 
 bool HTMLInputElement::shouldUseInputMethod() const
 {
-    return m_type == TEXT || m_type == SEARCH || m_type == ISINDEX;
+    return m_type == TEXT || m_type == SEARCH || m_type == ISINDEX || m_type == PASSWORD;
 }
 
 void HTMLInputElement::dispatchFocusEvent()
-- 
1.6.0.4

